code/datalab-app/package.json

```diff
@@ -44,7 +44,7 @@
     "winston": "2.3.1"
   },
   "devDependencies": {
-    "axios-mock-adapter": "1.9.0",
+    "axios-mock-adapter": "1.17.0",
```

Comment:
Why did these get bumped here? Did you need new functionality?

Reply:
This update is not actually not need, I can revert. It was update while I was hunting for an issue with the test. Will leave it upgraded unless there it conflicts with anything else.